openassessment/templates/openassessmentblock/oa_uploaded_file.html

```diff
@@ -0,0 +1,30 @@
+{% spaceless %}
+{% load i18n %}
+
+{% if file_upload_type %}
+    {% if header %}
+        <header class="{{ class_prefix }}__display__header">
+            <h3 class="{{ class_prefix }}__display__title">
+                {% trans header %}
+            </h3>
+        </header>
+    {% endif %}
+
+    <div class="{{ class_prefix }}__display__file {% if not file_url %}is--hidden{% endif %}" id="submission__{{ file_upload_type }}__upload" data-upload-type="{{ file_upload_type }}">
+        {% if file_upload_type == "image" %}
+            <img id="submission__answer__file"
+                 class="submission--image"
+                 alt="{% trans "The image associated with this submission." %}"
+                 src="{{ file_url }}" />
+        {% elif file_upload_type == "file" or file_upload_type == "custom" %}
+            <a href="{{ file_url }}" id="submission__answer__file" class="submission--file">
+                {% trans "The file associated with this submission." %}
+            </a>
+            {% if show_warning %}
+                <span>{% trans "(This file is uploaded by a user. Please use with caution.)" %}</span>
```

Comment:
I'd suggest,
Caution: This file was uploaded by another course learner. The file contents have not been validated.
However, I'd also like to check with some other folks at edX.